### DISCUSSION TITLE
WinNUT and NUT server for windows

### DISCUSSION THREAD

**Discussion by aceindy** [nutdotnet/WinNUT-Client#186] [Q&A] [answered]
Just found out that this client doesn't work with https://www.networkupstools.org/package/windows/NUT-Installer-2.6.5-6.msi
It tries to connect, but after the timeout it mentions disconnect. 
The logs reveal a authentication error
`Error while attempting to log in: INVALIDARGUMENT (ERR INVALID-ARGUMENT)
Query: LOGIN`
I had to fall back on the original https://github.com/gawindx/WinNUT-Client/releases for it to work

**Comment by gbakeman** [comment]
Could you please upload the full log file, set to Debug?

**Reply by aceindy** [reply]
Sure...there you go...
WinNUT Client - 2.2
Reconnection In Progress
Try 1 of 30
12/09/2024 17:25:00 [5396, WinNUT]: Update Icon
12/09/2024 17:25:00 [5396, UPS_Device]: Try Reconnect 1 / 30
12/09/2024 17:25:00 [5396, String]: New Log to CB_Current Log : Try Reconnect 1 / 30
12/09/2024 17:25:00 [5396, UPS_Device]: Beginning connection: sim@192.168.2.254:3493, Name: Eaton5PX2200 [AutoReconnect]
12/09/2024 17:25:00 [5396, Nut_Socket]: Attempting TCP socket connection to 192.168.2.254:3493...
12/09/2024 17:25:00 [5396, Nut_Socket]: Connection established and streams ready for 192.168.2.254:3493
12/09/2024 17:25:00 [5396, Nut_Socket]: Attempting authentication...
12/09/2024 17:25:00 [5396, Nut_Socket]: Error while attempting to log in: INVALIDARGUMENT (ERR INVALID-ARGUMENT)
Query: LOGIN
12/09/2024 17:25:00 [5396, Nut_Socket]: NUT server reports VER: Windows-v2.6.5-5-7-g72f380c NETVER: 1.2
12/09/2024 17:25:00 [5396, UPS_Device]: ups.realpower is not supported by server.
12/09/2024 17:25:00 [5396, UPS_Device]: Unhandled error while getting ups.realpower
12/09/2024 17:25:00 [5396, UPS_Device]: Using NominalPowerCalc method to calculate power usage.
12/09/2024 17:25:00 [5396, UPS_Device]: battery.capacity is not supported by server.
12/09/2024 17:25:00 [5396, UPS_Device]: Apply Fallback Value when retrieving battery.capacity
12/09/2024 17:25:00 [5396, WinNUT]: Eaton5PX2200 has indicated it's ready to start sending data.
12/09/2024 17:25:00 [5396, WinNUT]: Update Icon
12/09/2024 17:25:00 [5396, WinNUT]: NotifyIcon Text =>

I guess something has changed during the versions. The NUT server is now at 2.8, but the windows package is stuck at 2.6.5 😕 
But since I don't have a linux available, I am bound to a windows version 😞

**Reply by aceindy** [reply]
And just in case, the log from the original WinNUT (the one that works for me):
13/09/2024 10:41:05 Pid: 5420 WinNUT : Init Connexion to NutServer
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Connect To Nut Server
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter AuthLogin
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With ups.mfr : VAR Eaton5PX2200 ups.mfr "EATON"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With ups.model : VAR Eaton5PX2200 ups.model "Eaton 5PX 2200"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With ups.serial : VAR Eaton5PX2200 ups.serial "G095L47171"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With ups.firmware : VAR Eaton5PX2200 ups.firmware "14"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Connection to Nut Host Established
13/09/2024 10:41:05 Pid: 5420 String : New Log to CB_Current Log : Connection to Nut Host 192.168.2.254:3493 Established
13/09/2024 10:41:05 Pid: 5420 WinNUT : Update Icon
13/09/2024 10:41:05 Pid: 5420 WinNUT : NotifyIcon Text => 
WinNUT - 2.0
Connected
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter Retrieve_UPS_Data
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With output.frequency.nominal : VAR Eaton5PX2200 output.frequency.nominal "50.00"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With battery.charge : VAR Eaton5PX2200 battery.charge "100.00"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With battery.voltage : VAR Eaton5PX2200 battery.voltage "52.00"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With battery.runtime : VAR Eaton5PX2200 battery.runtime "2513.00"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Apply Fallback Value when retrieving battery.capacity
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With output.frequency : VAR Eaton5PX2200 output.frequency "50.00"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With input.frequency : VAR Eaton5PX2200 input.frequency "50.00"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Apply Fallback Value when retrieving input.voltage
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With output.voltage : VAR Eaton5PX2200 output.voltage "231.00"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With ups.load : VAR Eaton5PX2200 ups.load "12.00"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With ups.status : VAR Eaton5PX2200 ups.status "OL"
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Enter GetUPSVar
13/09/2024 10:41:05 Pid: 5420 UPS_Network : Process Result With ups.realpower.nominal : VAR Eaton5PX2200 ups.realpower.nominal "1980.00"
13/09/2024 10:41:05 Pid: 5420 WinNUT : Retrieve UPS Informations
13/09/2024 10:41:05 Pid: 5420 WinNUT : UPS is plugged
13/09/2024 10:41:05 Pid: 5420 WinNUT : Battery Charged
13/09/2024 10:41:05 Pid: 5420 WinNUT : Update Dial
13/09/2024 10:41:05 Pid: 5420 WinNUT : Update Icon
13/09/2024 10:41:05 Pid: 5420 WinNUT : Status Icon Changed
13/09/2024 10:41:05 Pid: 5420 WinNUT : New Icon Value For Systray : 1136
13/09/2024 10:41:05 Pid: 5420 WinNUT : New Icon Value For Gui : 1136
13/09/2024 10:41:05 Pid: 5420 WinNUT : NotifyIcon Text =>

**Reply by gbakeman** [reply, marked as answer]
Yeah, unfortunately the Windows build of NUT is quite old but I don't think you'll have to worry. I don't think you captured the entire log file, but from what I can see, you're missing several pre-release updates for WinNUT. Please download the latest pre-release [here](https://github.com/nutdotnet/WinNUT-Client/releases) and try again. If there's still issues, please upload the full log file after running a test.

**Reply by aceindy** [reply]
@gbakeman The pre-release seems to work ... great 😄 
13/09/2024 10:51:41 [4084, Logger]: Init log file: C:\Users\KM\AppData\Local\NUTDotNet\WinNUT Client\2.3.8988.30525\WinNUT-Client-2024-09-13.log
==== History of 22 previous events ====
[1] 13/09/2024 10:51:41 [4084, MyApplication]: WinNUT Client v2.3.8988.30525 starting up.
[2] 13/09/2024 10:51:41 [4084, MyApplication]: MyApplication_Startup complete.
[3] 13/09/2024 10:51:41 [4084, WinNUT]: NotifyIcons Initialised
[4] 13/09/2024 10:51:41 [4084, WinNUT]: Windows 10 Toast Notification Not Available. Required Version: 10.0.18362.0, Current: 6.1.7601.65536
[5] 13/09/2024 10:51:41 [4084, WinNUT]: Windows App Use Dark Theme
[6] 13/09/2024 10:51:41 [4084, WinNUT]: Windows Use Dark Theme
[7] 13/09/2024 10:51:41 [4084, WinNUT]: NotifyIcon Text => 
WinNUT Client - 2.3
Not Connected
[8] 13/09/2024 10:51:41 [4084, WinNUT]: Status Icon Changed
[9] 13/09/2024 10:51:41 [4084, WinNUT]: New Icon Value For Systray : 1216
[10] 13/09/2024 10:51:41 [4084, WinNUT]: New Icon Value For Gui : 1216
[11] 13/09/2024 10:51:41 [4084, WinNUT]: Update Icon at Startup
[12] 13/09/2024 10:51:41 [4084, WinNUT]: Beginning ApplyApplicationPreferences subroutine.
[13] 13/09/2024 10:51:41 [4084, WinNUT]: Initializing all display values and configurations.
[14] 13/09/2024 10:51:41 [4084, WinNUT]: Parameter Dial Input Voltage Need to be Updated
[15] 13/09/2024 10:51:41 [4084, WinNUT]: Parameter Dial Input Voltage Updated
[16] 13/09/2024 10:51:41 [4084, WinNUT]: Parameter Dial Input Frequency Need to be Updated
[17] 13/09/2024 10:51:41 [4084, WinNUT]: Parameter Dial Input Frequency Updated
[18] 13/09/2024 10:51:41 [4084, WinNUT]: Parameter Dial Output Voltage Need to be Updated
[19] 13/09/2024 10:51:41 [4084, WinNUT]: Parameter Dial Output Voltage Updated
[20] 13/09/2024 10:51:41 [4084, WinNUT]: Parameter Dial Voltage Battery Need to be Updated
[21] 13/09/2024 10:51:41 [4084, WinNUT]: Parameter Dial Voltage Battery Updated
[22] 13/09/2024 10:51:41 [4084, Logger]: Init log file: C:\Users\KM\AppData\Local\NUTDotNet\WinNUT Client\2.3.8988.30525\WinNUT-Client-2024-09-13.log
==== Begin Live Log ====
13/09/2024 10:51:41 [4084, WinNUT]: WinNut Preferences Applied.
13/09/2024 10:51:41 [4084, String]: New Log to CB_Current Log : WinNut Preferences Changed
13/09/2024 10:51:41 [4084, WinNUT]: WinNUT Form completed Load.
13/09/2024 10:51:41 [4084, WinNUT]: Main GUI activated.
13/09/2024 10:51:41 [4084, WinNUT]: Update Icon
13/09/2024 10:51:41 [4084, WinNUT]: WinNUT_Shown for the first time.
13/09/2024 10:51:41 [4084, WinNUT]: Update Icon
13/09/2024 10:51:41 [4084, WinNUT]: Show WinNut Main Gui
13/09/2024 10:51:44 [4084, WinNUT]: Open Pref Gui From Menu
13/09/2024 10:51:44 [4084, Pref_Gui]: Load Pref Gui
13/09/2024 10:51:44 [4084, WinNUT]: Main Gui Lose Focus
13/09/2024 10:51:44 [4084, WinNUT]: Update Icon
13/09/2024 10:51:44 [4084, Pref_Gui]: Pref Gui Opened.
13/09/2024 10:51:56 [4084, Pref_Gui]: Close Pref Gui from Button Cancel
13/09/2024 10:51:58 [4084, WinNUT]: Main GUI activated.
13/09/2024 10:51:58 [4084, WinNUT]: Update Icon
13/09/2024 10:52:00 [4084, WinNUT]: Open Pref Gui From Menu
13/09/2024 10:52:00 [4084, Pref_Gui]: Load Pref Gui
13/09/2024 10:52:00 [4084, WinNUT]: Main Gui Lose Focus
13/09/2024 10:52:00 [4084, WinNUT]: Update Icon
13/09/2024 10:52:00 [4084, Pref_Gui]: Pref Gui Opened.
13/09/2024 10:52:04 [4084, Pref_Gui]: Close Pref Gui from Button Cancel
13/09/2024 10:52:04 [4084, WinNUT]: Main GUI activated.
13/09/2024 10:52:04 [4084, WinNUT]: Update Icon
13/09/2024 10:52:06 [4084, WinNUT]: Menu_Persist checked state changing to True
13/09/2024 10:52:06 [4084, WinNUT]: SettingsPropertyChanged: NUT_AutoReconnect
13/09/2024 10:52:08 [4084, WinNUT]: Force Reconnect from menu
13/09/2024 10:52:08 [4084, WinNUT]: Running Client disconnect subroutine.
13/09/2024 10:52:08 [4084, WinNUT]: Attempted to disconnect when UPS_Device is Nothing.
13/09/2024 10:52:08 [4084, WinNUT]: Client UPS_Connect subroutine beginning.
13/09/2024 10:52:08 [4084, UPS_Device]: Beginning connection: sim@192.168.2.254:3493, Name: Eaton5PX2200 [AutoReconnect]
13/09/2024 10:52:08 [4084, Nut_Socket]: Attempting TCP socket connection to 192.168.2.254:3493...
13/09/2024 10:52:08 [4084, Nut_Socket]: Connection established and streams ready.
13/09/2024 10:52:08 [4084, Nut_Socket]: Gathering basic info about the NUT server...
13/09/2024 10:52:08 [4084, Nut_Socket]: Server version: Windows-v2.6.5-5-7-g72f380c
13/09/2024 10:52:08 [4084, Nut_Socket]: Protocol version: 1.2
13/09/2024 10:52:08 [4084, Nut_Socket]: Completed gathering basic info about NUT server.
13/09/2024 10:52:08 [4084, UPS_Device]: Retrieving basic UPS product information...
13/09/2024 10:52:08 [4084, UPS_Device]: Initializing other well-known UPS variables...
13/09/2024 10:52:08 [4084, UPS_Device]: output.current: 1
13/09/2024 10:52:08 [4084, UPS_Device]: output.voltage: 231
13/09/2024 10:52:08 [4084, UPS_Device]: output.power: 218
13/09/2024 10:52:08 [4084, UPS_Device]: Determining best method to calculate power usage...
13/09/2024 10:52:08 [4084, UPS_Device]: Using RealOutputPower method.
13/09/2024 10:52:08 [4084, UPS_Device]: battery.capacity is not supported by server.
13/09/2024 10:52:08 [4084, UPS_Device]: Apply Fallback Value when retrieving battery.capacity
13/09/2024 10:52:08 [4084, UPS_Device]: Completed retrieval of basic UPS product information.
13/09/2024 10:52:08 [4084, WinNUT]: Eaton5PX2200 has indicated it's ready to start sending data.
13/09/2024 10:52:08 [4084, WinNUT]: Update Icon
13/09/2024 10:52:08 [4084, WinNUT]: NotifyIcon Text => 
WinNUT Client - 2.3
Connected
13/09/2024 10:52:08 [4084, WinNUT]: Connection to Nut Host Established
13/09/2024 10:52:08 [4084, String]: New Log to CB_Current Log : Connection to Nut Host 192.168.2.254:3493 Established
13/09/2024 10:52:08 [4084, Nut_Socket]: Logging in to UPS [Eaton5PX2200] as user [sim] (Password provided)...
13/09/2024 10:52:08 [4084, Nut_Socket]: Authenticated successfully.
13/09/2024 10:52:15 [4084, WinNUT]: Open Pref Gui From Menu
13/09/2024 10:52:15 [4084, Pref_Gui]: Load Pref Gui
13/09/2024 10:52:15 [4084, WinNUT]: Main Gui Lose Focus
13/09/2024 10:52:15 [4084, WinNUT]: Update Icon
13/09/2024 10:52:15 [4084, Pref_Gui]: Pref Gui Opened.
13/09/2024 10:52:18 [4084, Pref_Gui]: User clicked ViewLog button.
13/09/2024 10:52:18 [4084, Pref_Gui]: Opened UI window to log location.
13/09/2024 10:52:19 [4084, UPS_Device]: Enter Retrieve_UPS_Datas
13/09/2024 10:52:19 [4084, UPS_Device]: input.voltage is not supported by server.
13/09/2024 10:52:19 [4084, UPS_Device]: Apply Fallback Value when retrieving input.voltage
13/09/2024 10:52:19 [4084, UPS_Device]: UPS statuses have CHANGED...
13/09/2024 10:52:19 [4084, UPS_Device]: Current statuses: OL
13/09/2024 10:52:19 [4084, WinNUT]: Handling new UPS status(es)...
13/09/2024 10:52:19 [4084, WinNUT]: Updating UPS data for Form.
13/09/2024 10:52:19 [4084, WinNUT]: Updating battery icons based on charge percent: 100%
13/09/2024 10:52:19 [4084, WinNUT]: Calculated estimated remaining battery time: 00:41:53
13/09/2024 10:52:19 [4084, WinNUT]: Update Dial
13/09/2024 10:52:19 [4084, WinNUT]: Update Icon
13/09/2024 10:52:19 [4084, WinNUT]: Status Icon Changed
13/09/2024 10:52:19 [4084, WinNUT]: New Icon Value For Systray : 1136
13/09/2024 10:52:19 [4084, WinNUT]: New Icon Value For Gui : 1136
13/09/2024 10:52:19 [4084, WinNUT]: NotifyIcon Text => 
WinNUT Client - 2.3
Connected
On Line
Battery OK
13/09/2024 10:52:19 [4084, WinNUT]: Battery Status => Charged
13/09/2024 10:52:29 [4084, UPS_Device]: Enter Retrieve_UPS_Datas
13/09/2024 10:52:29 [4084, UPS_Device]: input.voltage is not supported by server.
13/09/2024 10:52:29 [4084, UPS_Device]: Apply Fallback Value when retrieving input.voltage
13/09/2024 10:52:29 [4084, UPS_Device]: UPS statuses have not changed since last update, skipping.
13/09/2024 10:52:29 [4084, WinNUT]: Updating UPS data for Form.
13/09/2024 10:52:29 [4084, WinNUT]: Updating battery icons based on charge percent: 100%
13/09/2024 10:52:29 [4084, WinNUT]: Calculated estimated remaining battery time: 00:41:53
13/09/2024 10:52:29 [4084, WinNUT]: Update Dial
13/09/2024 10:52:29 [4084, WinNUT]: Update Icon
13/09/2024 10:52:29 [4084, WinNUT]: NotifyIcon Text => 
WinNUT Client - 2.3
Connected
On Line
Battery OK
13/09/2024 10:52:29 [4084, WinNUT]: Battery Status => Charged
13/09/2024 10:52:29 [4084, Pref_Gui]: Save Parameters.
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: NUT_ServerAddress
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: NUT_ServerPort
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: NUT_UPSName
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: NUT_PollIntervalMsec
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: NUT_Username
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: NUT_Password
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: NUT_AutoReconnect
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CAL_VoltInMin
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CAL_VoltInMax
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CAL_FreqInNom
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CAL_FreqInMin
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CAL_FreqInMax
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CAL_VoltOutMin
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CAL_VoltOutMax
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CAL_BattVMin
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CAL_BattVMax
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: MinimizeToTray
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: MinimizeOnStart
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: CloseToTray
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: StartWithWindows
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: LG_LogToFile
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: LG_LogLevel
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: PW_BattChrgFloor
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: PW_RuntimeFloor
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: PW_Immediate
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: PW_RespectFSD
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: PW_StopType
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: PW_StopDelaySec
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: PW_UserExtendStopTimer
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: PW_ExtendDelaySec
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: UP_CheckAtStart
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: UP_AutoChkDelay
13/09/2024 10:52:29 [4084, WinNUT]: SettingsPropertyChanged: UP_Branch
13/09/2024 10:52:29 [4084, WinNUT]: Beginning ApplyApplicationPreferences subroutine.
13/09/2024 10:52:29 [4084, WinNUT]: Running Client disconnect subroutine.
13/09/2024 10:52:29 [4084, UPS_Device]: Processing request to disconnect...
13/09/2024 10:52:29 [4084, WinNUT]: Initializing all display values and configurations.
13/09/2024 10:52:29 [4084, WinNUT]: Update Icon
13/09/2024 10:52:29 [4084, WinNUT]: Status Icon Changed
13/09/2024 10:52:29 [4084, WinNUT]: New Icon Value For Systray : 1216
13/09/2024 10:52:29 [4084, WinNUT]: New Icon Value For Gui : 1216
13/09/2024 10:52:29 [4084, WinNUT]: NotifyIcon Text => 
WinNUT Client - 2.3
Not Connected
13/09/2024 10:52:29 [4084, WinNUT]: Battery Status => Unknown
13/09/2024 10:52:29 [4084, WinNUT]: Disconnected from Nut Host
13/09/2024 10:52:29 [4084, String]: New Log to CB_Current Log : Disconnected from Nut Host
13/09/2024 10:52:29 [4084, WinNUT]: Client UPS_Connect subroutine beginning.
13/09/2024 10:52:29 [4084, UPS_Device]: Beginning connection: sim@192.168.2.254:3493, Name: Eaton5PX2200 [AutoReconnect]
13/09/2024 10:52:29 [4084, Nut_Socket]: Attempting TCP socket connection to 192.168.2.254:3493...
13/09/2024 10:52:29 [4084, Nut_Socket]: Connection established and streams ready.
13/09/2024 10:52:29 [4084, Nut_Socket]: Gathering basic info about the NUT server...
13/09/2024 10:52:29 [4084, Nut_Socket]: Server version: Windows-v2.6.5-5-7-g72f380c
13/09/2024 10:52:29 [4084, Nut_Socket]: Protocol version: 1.2
13/09/2024 10:52:29 [4084, Nut_Socket]: Completed gathering basic info about NUT server.
13/09/2024 10:52:29 [4084, UPS_Device]: Retrieving basic UPS product information...
13/09/2024 10:52:29 [4084, UPS_Device]: Initializing other well-known UPS variables...
13/09/2024 10:52:29 [4084, UPS_Device]: output.current: 1
13/09/2024 10:52:29 [4084, UPS_Device]: output.voltage: 231
13/09/2024 10:52:29 [4084, UPS_Device]: output.power: 218
13/09/2024 10:52:29 [4084, UPS_Device]: Determining best method to calculate power usage...
13/09/2024 10:52:29 [4084, UPS_Device]: Using RealOutputPower method.
13/09/2024 10:52:29 [4084, UPS_Device]: battery.capacity is not supported by server.
13/09/2024 10:52:29 [4084, UPS_Device]: Apply Fallback Value when retrieving battery.capacity
13/09/2024 10:52:29 [4084, UPS_Device]: Completed retrieval of basic UPS product information.
13/09/2024 10:52:29 [4084, WinNUT]: Eaton5PX2200 has indicated it's ready to start sending data.
13/09/2024 10:52:29 [4084, WinNUT]: Update Icon
13/09/2024 10:52:29 [4084, WinNUT]: NotifyIcon Text => 
WinNUT Client - 2.3
Connected
13/09/2024 10:52:29 [4084, WinNUT]: Connection to Nut Host Established
13/09/2024 10:52:29 [4084, String]: New Log to CB_Current Log : Connection to Nut Host 192.168.2.254:3493 Established
13/09/2024 10:52:29 [4084, Nut_Socket]: Logging in to UPS [Eaton5PX2200] as user [sim] (Password provided)...
13/09/2024 10:52:29 [4084, Nut_Socket]: Authenticated successfully.
13/09/2024 10:52:29 [4084, WinNUT]: WinNut Preferences Applied.
13/09/2024 10:52:29 [4084, String]: New Log to CB_Current Log : WinNut Preferences Changed
13/09/2024 10:52:29 [4084, Pref_Gui]: Preferences Saved
13/09/2024 10:52:29 [4084, WinNUT]: Main GUI activated.
13/09/2024 10:52:29 [4084, WinNUT]: Update Icon
13/09/2024 10:52:35 [4084, WinNUT]: Open Pref Gui From Menu
13/09/2024 10:52:35 [4084, Pref_Gui]: Load Pref Gui
13/09/2024 10:52:35 [4084, WinNUT]: Main Gui Lose Focus
13/09/2024 10:52:35 [4084, WinNUT]: Update Icon
13/09/2024 10:52:35 [4084, Pref_Gui]: Pref Gui Opened.
13/09/2024 10:52:37 [4084, Pref_Gui]: User clicked ViewLog button.
13/09/2024 10:52:37 [4084, Pref_Gui]: Opened UI window to log location.

**Reply by gbakeman** [reply]
Excellent, glad it worked!

**Comment by aceindy** [comment]
Works with latest pre-release
